assets/src/edit-story/components/panels/preset/presetPanel.js

```diff
@@ -90,7 +90,7 @@ function PresetPanel({
     [colors, isColor, textStyles, updateStory]
   );
 
-  const hasPresets = colors.length > 0;
+  const hasPresets = colors.length > 0 || textStyles.length > 0;
```

Comment:
Should we dot this instead?

```suggestion
  const hasPresets = 'color' === presetType ? colors.length > 0 : textStyles.length > 0;
```

Otherwise the effect below to exit edit mode when there are no presets left doesn't work.

Could be tested easily I think.

Reply:
Yep, good point, `presets` as the set of relevant presets is used below as well actually for header, was thinking of perhaps doing something like
`const presets = isColor ? colors : textStyles` as is used below and then we could use just `presets.length`.